JSONPOG updating in setup.sh

Chihwan-An opened the pull request:
Checking JSONPOG repository latest version with not using hash , checking committs .

Chihwan-An commented:
I didn't meant to close PR, there was some confusing about managing git. 

I pushed my modified files to forked registory , before updating latest version in upstream.
This made some error and I thought I corrected error and everything is ok , I uploaded the 
"setup.sh" below.

But thanks to comment I just force pushed with latest version to my fork repo , which made PR close automatically.

choij1589 commented on setup.sh:
```diff
@@ -126,25 +126,32 @@ echo "@@@@ Correction include: $CORRECTION_INCLUDE_DIR"
+
+    if [ "$BEHIND" -gt 0 ]; then
+        echo "@@@@ Repository is $BEHIND commits behind origin/master, updating..."
+        git merge origin/master
```
Can you add logic for accept / reject the update? It is hard to track the changes if the jsonpog repo is auto-updating for every setup, especially for reproducing results or debugging the issue.

Chihwan-An commented:
v3 , 
- Added current Local & jsonpog commit and hash tag which could make easy to track.
- Added Y/N choosing not to update jsonpogrepo to latest version.
